lib/appc/annotation.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 The appc Authors
```

Comment:
Because more things are singular than plural, and something something consistency. I'll singular-ize dependencies too.